pymatgen/analysis/eos.py

```diff
@@ -1,96 +1,88 @@
 # coding: utf-8
```

Comment:
@matk86  @shyuep 

These changes in eos.py have not been discussed and, most importantly, they break 
the previous API provided by EOS.

This piece of code is used in our automatic flows for the generation of the pseudopotentials 
and I've already spent hours to port the other projects we are developing to pmg4.

Now we have reached pmg4.3.1 and I think we should try to avoid drastic changes in the API, 
especially in parts that are used by other members of the community.

Improvements and bug fixes in eos.py are welcome but this is not the correct approach.
If you don't like the API or you want to refactor the code, you should try to maintain compatibility with
the previous API, deprecate methods or, if really needed, replace the old code with another one.

Could you please revert eos.py to the original version and include your fixes/improvements 
without breaking the API?
Thanks
Matteo